### ISSUE TITLE
Bug 58069179: [React Native Gallery -> Sensitive Info]: The text present in the edit fields are getting truncated after applying text scaling to 200%.

### ISSUE THREAD

**Issue by vineethkuttan** [microsoft/react-native-gallery#608]
**Build Details:**
OS Version: Dev (OS Build 27871.1000)
App: React Native Gallery 
App Version: 1.0.18.0
React Native Windows Version: 0.79.0

AT: Text Scaling  
Resolution: 1280*768

**Pre-Requisite:**
Text Scaling (Settings->Accessibility->Text size-> Drag the slider to 200% ->Apply))

**Repro-Steps:**
Launch the React Native Gallery app.
Navigate to ‘Status & Info' button present in left navigation pane and expand it.
Navigate to 'Sensitive Info' button and activate it.
After applying text scaling, observe the text present in the edit fields.
Actual Result:
The text present in the edit fields are getting truncated after applying text scaling to 200%.

**Expected Result:**
The text present in the edit fields should be visible clearly to the user without any data loss and truncation.

**User impact:**

Users with low vision will not be able to consume the content of the button if it is not visible in Text scaling mode.

**MAS Reference:**
[MAS 1.4.4 – Resize Text](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7b5BE8E205-7508-414C-A273-657770321DA0%7d&file=MAS%201.4.4%20%E2%80%93%20Resize%20Text.docx&wdLOR=cD434D0F2-F515-4FD8-B4DA-BA99282968ED&action=default&mobileredirect=true) 

**Trap ID:**
[7.1 - Data Loss](https://aka.ms/Data_Loss)

"Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”

**Comment by chiaramooney** [comment]
This change should only be a JS side change (no Paper architecture specific knowledge needed). Similar to 200% errors we saw in new arch gallery.

Note: The source code for the Paper React Native Gallery begins at the root of the repo rather than inside the New Arch folder.

**Comment by vineethkuttan** [comment]
> > Not Applicable for new gallery , as for now.
> 
> Not applicable for new gallery but must be fixed for old gallery otherwise accessibility grading score will drop out of compliance for old architecture.
@sreehari-rajan Please have look on this.
